packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -54,6 +57,73 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   return matrix;
 }
 
+bool _extensionsInitialized = false;
+
+void initServiceExtensions() {
```

Comment:
We could add a guard for checked mode. But note that the service extensions would then not be available to a debugger, when debugging an app that's running in production mode.

Happy to add the code in an `assert`. We can move it out to again if necessary.